src/SFA.DAS.ProviderRegistrations/Application/Queries/GetProviderByUkprn/GetProviderByUkprnQueryResult.cs

```diff
@@ -0,0 +1,14 @@
+using SFA.DAS.Apprenticeships.Api.Types.Providers;
+
+namespace SFA.DAS.ProviderRegistrations.Application.Queries.GetProviderByUkprn
+{
+    public class GetProviderByUkprnQueryResult
+    {
+        public GetProviderByUkprnQueryResult(Provider provider)
```

Comment:
I don't think I like the Result object having a dependency on the API type

Reply:
Changed